htdocs/takepos/index.php

```diff
@@ -1012,10 +1016,10 @@ function WeighingScale(){
 if (empty($paiementsModes)) {
 	$langs->load('errors');
 	setEventMessages($langs->trans("ErrorModuleSetupNotComplete", $langs->transnoentitiesnoconv("TakePOS")), null, 'errors');
-	setEventMessages($langs->trans("ProblemIsInSetupOfTerminal", $_SESSION["takeposterminal"]), null, 'errors');
+	setEventMessages($langs->trans("ProblemIsInSetupOfTerminal", isset($_SESSION["takeposterminal"]) ? $_SESSION["takeposterminal"] : ''), null, 'errors');
```

Comment:
using parenthesis will lead to more secured code

Reply:
ok, all occurences changed :)